packages/statverif/statverif.1.97pl1.2/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+name: "statverif"
+version: "1.97pl1.2"
+maintainer: "Eike Ritter <E.Ritter@cs.bham.ac.uk>"
+authors: ["Eike Ritter, Joshua Philipps, Bruno Blanchet, Vincent Cheval, and Marc Sylvestre "]
+homepage: "http://sec.cs.bham.ac.uk/research/StatVerif"
+bug-reports: "E.Ritter@cs.bham.ac.uk"
+license: "GPL 2"
+dev-repo: "git+https://github.com/rittere/statverif.git"
+build: [
+   [make]
+]
+install: [make "prefix=%{prefix}%" "install"]
+remove: [["ocamlfind" "remove" "statverif"] ["rm" "-f" "%{bin}%/statverif"] ["rm" "-f" "%{bin}%/statveriftotex"]]
```

Comment:
```suggestion
```
The remove field is unnecessary since opam 2.0

Reply:
Done.